.travis.yml

```diff
@@ -56,7 +56,7 @@ matrix:
             - llvm-toolchain-precise 
 
 before_install:
-  - git clone --depth=1 --branch=dev_nana_in_examples https://github.com/qPCR4vir/nana-demo.git nana-demo
+  - git clone --depth=1 --branch=dev_nana_fs https://github.com/qPCR4vir/nana-demo.git nana-demo
```

Comment:
Does it mean that `nana/develop` branch uses different `dev_nana_*` branch of `nana-demo` in different development stages?
I found a description in your `nana-demo/dev_nana_fs` branch.
> nana/develope - nana/stable.

This is a bit weird. IMO, becuase `nana/develop` is unstable. Perhaps, the `nana-demo/stable` is more suited for `nana/master` and `nana/hotfix`.